graylog2-web-interface/src/components/content-packs/ContentPackApplyParameter.css

```diff
@@ -1,3 +1,8 @@
 :local(.applyParameter) {
     margin-top: 8px;
 }
+
+:local(.applyTable) {
```

Comment:
Is it possible to use styled-components here instead of adding more to CSS files?